### PR TITLE
fix: Support for updating the extension component in vue page

### PR DESCRIPTION
**Pull request by carvinlo**
# New Pull Request

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

Please make sure you can check the following boxes:

- [x] I have read the **CONTRIBUTING** document
- [ ] My code follows the code style of this project
- [ ] All new and existing tests passed

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes

### Description

When the count in the react component increases, the react extension will be updated. But in the vue component increases, the extension will not be updated.

### Remarks

[Optionally place any follow-up comments, remarks, observations, or notes here for future reference]

### CODE REVIEW

File: src/framework/piral-core/src/modules/api.ts [hunk 1]
```diff
@@ -111,8 +111,7 @@ export function createCoreApi(context: GlobalStateContext): PiletApiExtender<Pil
         context.unregisterExtension(name as string, arg);
       },
       renderHtmlExtension(element, props) {
-        const [dispose] = render(context, element, props);
-        return dispose;
+        return render(context, element, props);
```
Comment (FlorianRappl) [inline src/framework/piral-core/src/modules/api.ts:114]: This is a breaking change - we will not do that. Instead, use the DOM event to propagate the update (extension-props-changed).

**Review by FlorianRappl — COMMENTED** [review]
This seems to use an old commit base of `develop`. Can you update your branch? Already some work was done in that regard. Please first check if that satisfies your needs / what is missing there. Thanks!

File: src/framework/piral-core/src/modules/api.ts (hunk 1 above)
Comment (carvinlo) [reply src/framework/piral-core/src/modules/api.ts:114]: Ok👌

**Comment by carvinlo** [comment]
It seems to be working fine for me🎉. [vue sample](https://github.com/carvinlo/piral/tree/amu/develop-vue-sample)

**Review by FlorianRappl — APPROVED** [review]
Great! Thanks for testing and improving / fixing! :beers:

**Pull request merged**